arch/X86/X86GenAsmWriter1.inc

```diff
@@ -9,6 +9,8 @@
 /* Capstone Disassembly Engine, http://www.capstone-engine.org */
 /* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2015 */
 
+#ifndef CAPSTONE_DIET
```

Comment:
I do not see this function filling cs_detail in any way but formatting the already parsed argument in a printable form. Are you sure?